Example doesn't build

The example supplied in the template doesn't build. In mciantyre/teensy4-rs#47, we changed how the BSP exposes its LED, since the GPIO is also used for SPI CLK.

Reproduce the bug following

```
cargo generate --git https://github.com/mciantyre/teensy4-rs-template --name hello-world
cd hello-world
cargo check

error[E0609]: no field `led` on type `teensy4_bsp::Peripherals`
  --> src/main.rs:16:21
   |
16 |         peripherals.led.set_high().unwrap();
   |                     ^^^ unknown field
   |
   = note: available fields are: `ccm`, `pit`, `usb`, `dcdc`, `pwm1` ... and 8 others

warning: unused import: `embedded_hal::digital::v2::OutputPin`
  --> src/main.rs:10:5
   |
10 | use embedded_hal::digital::v2::OutputPin;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default

error: aborting due to previous error

For more information about this error, try `rustc --explain E0609`.
error: could not compile `hello-world`.
```

--- a/.genignore
+++ b/.genignore
@@ -2,4 +2,4 @@ README.md
 LICENSE-APACHE
 LICENSE-MIT
 Cargo.lock
-.github/*
+.github/workflows/cron.yml

--- a/.genignore
+++ b/.genignore
@@ -2,3 +2,4 @@ README.md
 LICENSE-APACHE
 LICENSE-MIT
 Cargo.lock
+.github/*

--- a/.genignore
+++ b/.genignore
@@ -2,4 +2,4 @@ README.md
 LICENSE-APACHE
 LICENSE-MIT
 Cargo.lock
-.github/workflows/cron.yml
+.github

--- a/.github/workflows/cron.yml
+++ b/.github/workflows/cron.yml
@@ -4,6 +4,8 @@ on:
   schedule:
     # At 15:28 on a Tuesday
     - cron: "28 15 * * 2"
+  pull_request:
+    branches: [ master ]
 
 jobs:
   build:

--- a/.github/workflows/cron.yml
+++ b/.github/workflows/cron.yml
@@ -16,6 +16,7 @@ jobs:
       - name: Install thumbv7em-none-eabihf target
         uses: actions-rs/toolchain@v1
         with:
+          toolchain: stable
           target: thumbv7em-none-eabihf
           override: true
       - name: Install cargo-generate

--- a/.github/workflows/cron.yml
+++ b/.github/workflows/cron.yml
@@ -2,10 +2,8 @@ name: weekly
 
 on:
   schedule:
-    # At 15:28 on a Tuesday
-    - cron: "28 15 * * 2"
-  pull_request:
-    branches: [ master ]
+    # At 15:28 on a Thursday
+    - cron: "28 15 * * 4"
 
 jobs:
   build:

--- a/.github/workflows/cron.yml
+++ b/.github/workflows/cron.yml
@@ -1,0 +1,19 @@
+name: weekly
+
+on:
+  schedule:
+    # At 15:28 on a Tuesday
+    - cron: "28 15 * * 2"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install build dependencies
+        run: sudo apt-get install -y gcc-arm-none-eabi binutils-arm-none-eabi libnewlib-arm-none-eabi
+      - name: Install cargo-generate
+        run: cargo install cargo-generate
+      - name: Generate the template
+        run: cargo generate --git https://github.com/mciantyre/teensy4-rs-template --name hello-world
+      - name: Check the template
+        run: cd hello-world && cargo check --verbose

--- a/.github/workflows/cron.yml
+++ b/.github/workflows/cron.yml
@@ -13,9 +13,16 @@ jobs:
     steps:
       - name: Install build dependencies
         run: sudo apt-get install -y gcc-arm-none-eabi binutils-arm-none-eabi libnewlib-arm-none-eabi
+      - name: Install thumbv7em-none-eabihf target
+        uses: actions-rs/toolchain@v1
+        with:
+          target: thumbv7em-none-eabihf
+          override: true
       - name: Install cargo-generate
         run: cargo install cargo-generate
       - name: Generate the template
         run: cargo generate --git https://github.com/mciantyre/teensy4-rs-template --name hello-world
       - name: Check the template
+        # Not including --target flag; template should support a .cargo/config
+        # with the correct target.
         run: cd hello-world && cargo check --verbose

--- a/README.md
+++ b/README.md
@@ -14,3 +14,5 @@ README](https://github.com/mciantyre/teensy4-rs/blob/master/README.md) for build
 Use the `hardware-test.sh` script to build and prepare a release build. If the
 `teensy_cli_loader` is on your `$PATH`, we'll use it to load the binary on your
 Teensy 4. The example provided in the template should turn on your Teensy 4's LED.
+
+Weekly build: ![weekly build](https://github.com/mciantyre/teensy4-rs-template/workflows/weekly/badge.svg)

--- a/src/main.rs
+++ b/src/main.rs
@@ -12,8 +12,10 @@ use embedded_hal::digital::v2::OutputPin;
 #[entry]
 fn main() -> ! {
     let mut peripherals = bsp::Peripherals::take().unwrap();
+    let mut led = bsp::configure_led(&mut peripherals.gpr, peripherals.pins.p13);
+
     loop {
-        peripherals.led.set_high().unwrap();
+        led.set_high().unwrap();
         wfi();
     }
 }

